### PR TITLE
hide and disable the table selection inputs

### LINKED ISSUE #136: Table option to suppress checkbox column
I see many uses of Table to just view the results in a table format without the need to have the table drive further down-selection. Although the checkboxes on the left are great for having the table drive the next steps (selected rows) it would be great to have an option to have the table without this column.

Perhaps an option like 'selectable' with a default of true?

### PR DESCRIPTION
**Pull request by Fil**
closes #136

### CODE REVIEW

File: src/table.js [hunk 1]
```diff
@@ -96,8 +97,8 @@ function initialize(
   let anchor = null, head = null;
 
   const tbody = html`<tbody>`;
-  const tr = html`<tr><td><input type=${multiple ? "checkbox" : "radio"} name=${multiple ? null : "radio"}></td>${columns.map(() => html`<td>`)}`;
-  const theadr = html`<tr><th><input type=checkbox onclick=${reselectAll} disabled=${!multiple}></th>${columns.map((column) => html`<th title=${column} onclick=${event => resort(event, column)}><span></span>${header && column in header ? header[column] : column}</th>`)}</tr>`;
+  const tr = html`<tr>${selectable ? html`<td><input type=${multiple ? "checkbox" : "radio"} name=${multiple ? null : "radio"}>` : html`<td style="width:0"><span>`}${columns.map(() => html`<td>`)}`;
+  const theadr = html`<tr>${selectable ? html`<th><input type=checkbox onclick=${reselectAll} disabled=${!multiple}>` : html`<th style="width:0"><span>`}${columns.map((column) => html`<th title=${column} onclick=${event => resort(event, column)}><span></span>${header && column in header ? header[column] : column}</th>`)}</tr>`;
```
Comment (mbostock) [inline src/table.js:101]: What’s the empty `<span>` for?

File: src/table.js [hunk 2]
```diff
@@ -125,9 +126,11 @@ function initialize(
   function appendRow(d, i) {
     const itr = tr.cloneNode(true);
     const input = inputof(itr);
-    input.onclick = reselect;
-    input.checked = selected.has(i);
-    input.value = i;
+    if (selectable) {
```
Comment (mbostock) [inline src/table.js:129]: Could we check if input is null here instead? Or should we check selectable before calling inputof?

I’m guessing there are other places we need to check if inputof returns null, such as reinput which is called on resort.

File: src/table.js [hunk 3]
```diff
@@ -54,6 +54,7 @@ function initialize(
     rows = 11.5, // maximum number of rows to show
     width = {}, // object of column name to width, or overall table width
     multiple = true,
+    select: selectable = true, // is the table selectable?
```
Comment (CobusT) [inline src/table.js:57]: I find `selectable` a bit more descriptive than `select` as an option. Thoughts?

**Comment by CobusT** [comment]
Do we normally include the changes to the documentation (README) to these PRs or are those done separately?

File: src/table.js (hunk 2 above)
Comment (Fil) [reply src/table.js]: yes that's better than adding the empty span: PR updated

File: src/table.js (hunk 3 above)
Comment (Fil) [reply src/table.js:57]: I have no preference; in the code I've been using selectable because the name is already used by the select function.

**Comment by mbostock** [comment]
Merged into #262.

**Pull request closed without merging**